src/renderer/sw_engine/tvgSwRle.cpp

```diff
@@ -322,7 +322,7 @@ static void _genSpan(SwRleData* rle, const SwSpan* spans, uint32_t count)
 }
 
 
-static void _horizLine(RleWorker& rw, SwCoord x, SwCoord y, SwCoord area, SwCoord acount)
+static void _horizLine(RleWorker& rw, SwCoord x, SwCoord y, SwCoord area, SwCoord aCount)
```

Comment:
This patch is incorrect.

`acount` -> `account`

Reply:
I think it's really `count` or `offset` and not an _account_. My default correction would have been to `account` because that's usually right...

Reply:
agree, thanks.